sktime/utils/plotting.py

```diff
@@ -109,3 +109,70 @@ def format_fn(tick_val, tick_pos):
         ax.legend()
 
     return fig, ax
+
+
+def plot_lags(series, lags=1):
+    """Plot one or more lagged versions of a time series
+
+    Parameters
+    ----------
+    series : pd.Series
+        Time series for plotting lags
+    lags : int or array-like
+        Number of lags to plot.
+        int         - plot the specified lag
+        array-like  - plot specified lags in the array/list
+
+    Returns
+    -------
+
+    fig   :  plt.Figure
+    axes  :  plt.Axis or ndarray of plt.Axis objects
+
+    Example
+    -------
+
+    Given the following time series
+
```

Comment:
Need to update example to pass doctests. I'd also use an example dataset built into sktime (look into `load_airline()`).